Polygon/include/CGAL/Multipolygon_with_holes_2.h

```diff
@@ -72,15 +72,28 @@ class Multipolygon_with_holes_2 {
 
   Polygon_with_holes_iterator polygons_with_holes_end() { return m_polygons.end(); }
 
+  Polygon_with_holes_iterator begin() { return m_polygons.begin(); }
+
+  Polygon_with_holes_iterator end() { return m_polygons.end(); }
+
   Polygon_with_holes_const_iterator polygons_with_holes_begin() const { return m_polygons.begin(); }
 
   Polygon_with_holes_const_iterator polygons_with_holes_end() const { return m_polygons.end(); }
 
+  Polygon_with_holes_const_iterator begin() const { return m_polygons.begin(); }
+
+  Polygon_with_holes_const_iterator end() const { return m_polygons.end(); }
+
+
   void add_polygon(const Polygon_2& pgn) { m_polygons.push_back(Polygon_with_holes_2(pgn)); }
 
+  void add_polygon(Polygon_2&& pgn) { m_polygons.push_back(Polygon_with_holes_2(std::forward<Polygon_with_holes_2>(pgn))); }
```

Comment:
`emplace_back` can avoid creating a temporary `Polygon_with_holes_2`, and will create it directly inplace in the container.

```suggestion
  void add_polygon(Polygon_2&& pgn) { m_polygons.emplace_back(std::move(pgn)); }
```